modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -99,9 +100,9 @@ Returning to the Good Weather use case, imagine that the company uses the follow
 
 image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
 
-As shown in the diagram, you first configure your external client provider and assign it to the appropriate business groups and environments. When you create or modify an API within a specific business group or environment, the new client provider is displayed. You can reselect the appropriate client provider. 
+As shown in the diagram, you first configure your external client provider and assign it to the appropriate environments. When you create or modify an API within a specific environment, the new client provider is displayed. You can reselect the appropriate client provider. 
 
-When you promote an API to another environment, the target environment must have the client provide configured as in the source environment. Otherwise, the promotion does not complete successfully. 
+When you promote an API to another environment, the target environment must have the client provider configured as in the source environment. Otherwise, a promoted API's IdP policy may not match the API's associated IdP. To resolve this mismatch, delete the IDP policy. 
```

Comment:
```suggestion
When you promote an API to another environment, the target environment must have the client provider configured as in the source environment. Otherwise, a promoted API's IdP policy might not match the API's associated IdP. To resolve this mismatch, delete the IDP policy. 
```